l10n_ch_invoice_reports/data/reports.xml

```diff
@@ -0,0 +1,65 @@
+    <record id="account_move_isr_report" model="ir.actions.report">
+        <field name="name">Invoice with ISR</field>
+        <field name="model">account.move</field>
+        <field name="report_type">qweb-pdf</field>
+        <field
+            name="report_name"
+        >l10n_ch_invoice_reports.invoice_isr_report_main</field>
+        <field
+            name="report_file"
+        >l10n_ch_invoice_reports.invoice_isr_report_main</field>
+        <field name="print_report_name">'Invoice-ISR-%s' % object.name</field>
+        <field name="attachment_use" eval="False" />
+        <field name="attachment" />
+        <field name="binding_model_id" ref="account.model_account_move" />
+        <field name="binding_type">report</field>
+    </record>
+
+    <record id="account_move_qr_report" model="ir.actions.report">
+        <field name="name">Invoice with QR</field>
+        <field name="model">account.move</field>
+        <field name="report_type">qweb-pdf</field>
+        <field name="report_name">l10n_ch_invoice_reports.invoice_qr_report_main</field>
+        <field name="report_file">l10n_ch_invoice_reports.invoice_qr_report_main</field>
+        <field name="print_report_name">'Invoice-QR-%s' % object.name</field>
+        <field name="attachment_use" eval="False" />
+        <field name="attachment" />
+        <field name="binding_model_id" ref="account.model_account_move" />
+        <field name="binding_type">report</field>
+    </record>
+
+    <record id="account_move_qr_isr_report" model="ir.actions.report">
+        <field name="name">Invoice with QR and ISR</field>
+        <field name="model">account.move</field>
+        <field name="report_type">qweb-pdf</field>
+        <field
+            name="report_name"
+        >l10n_ch_invoice_reports.invoice_qr_isr_report_main</field>
+        <field
+            name="report_file"
+        >l10n_ch_invoice_reports.invoice_qr_isr_report_main</field>
+        <field name="print_report_name">'Invoice-QR-ISR-%s' % object.name</field>
+        <field name="attachment_use" eval="False" />
+        <field name="attachment" />
+        <field name="binding_model_id" ref="account.model_account_move" />
+        <field name="binding_type">report</field>
+    </record>
+
+    <record id="account_move_payment_report" model="ir.actions.report">
+        <field name="name">Invoice with payment slip(s)</field>
+        <field name="model">account.move</field>
+        <field name="report_type">qweb-pdf</field>
+        <field
+            name="report_name"
+        >l10n_ch_invoice_reports.account_move_payment_report</field>
+        <field
+            name="report_file"
+        >l10n_ch_invoice_reports.account_move_payment_report</field>
+        <field name="print_report_name">('invoice-payment-slip_'+object.name)</field>
+        <field name="attachment">''invoice_'+object.name+'_with_payslip.pdf''</field>
+        <field name="attachment_use">True</field>
+    </record>
```

Comment:
@sonhd91  did you try to run this code on local odoo instance and print report?

Reply:
I did, it's printout the successfully

Reply:
well, i tried to test your PR few days back and applied locally change above (to get rid of old reports) and had issue with view id not found when tried to print invoice. I opened parallel migration PR yesterday (since we need this working on v15 for our customers) and described it there since i faced it again:

https://github.com/OCA/l10n-switzerland/pull/673#issuecomment-1255154437

that's why i am suprised it works in your case.

Reply:
I just updated branch, this time I add `binding_model_id` in xml file for the report show up on the print button. Could you please check again ?